docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -226,3 +229,29 @@ and a file system location of ``/var/my_tables/test_table``::
         format = 'PARQUET',
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
+
+.. _iceberg-materialized-views:
+
+Materialized views
+^^^^^^^^^^^^^^^^^^
+
+The Iceberg connector supports :ref:`sql-materialized-views-management`. In the
+underlying system each materialized view consists of a view definition and an
+Iceberg storage table. The storage table name is stored as a materialized view
+property. The data is stored in that storage table.
+
+You can use the :ref:`iceberg-table-properties` to control the created storage
+table and therefore the layout and performance. For example, you can use the
+following clause with :doc:`/sql/create-materialized-view` to use the ORC format
+for the data files and partition the storage per day using the column
+``_date``::
+
+    WITH ( format = 'ORC', partitioning = ARRAY['_date'] )
+
+Updating the data in the materialized view with
+:doc:`/sql/refresh-materialized-view` deletes the data from the storage table,
+and inserts new data that is the result of executing the materialized view
+query.
```

Comment:
@sopel39 pease confirm.

Also, this could be understood as "during REFRESH, users may temporarily get zero rows results when querying the materialized view". We should be clear whether this may or may not be the case.

Reply:
> @sopel39 pease confirm.

Yes, in `finishRefreshMaterializedView` we first drop data from storage table and then commit new data

Reply:
@mosabua please add a warning (or sth) about this in the docs.

Reply:
I don't think a warning is warranted as @anjalinorwood approved the PR as is.. it seems like a very minor detail with potentially no user impact. I dont know how it works out in practical terms either. E.g. if refresh takes long to populate the table. Is this really a problem or is access to the materialized view blocked during the refresh somehow?

What is your experience with lots of users on this @anjalinorwood ?

Reply:
We discussed and agreed that a warning is appropriate. It looks like it would be easy to change the code to do this as a single atomic Iceberg transaction, so we can then remove the warning.